pkg/operator/ceph/disruption/machinedisruption/reconcile.go

```diff
@@ -123,31 +131,27 @@ func (r *MachineDisruptionReconciler) Reconcile(request reconcile.Request) (reco
 	// Check if the cluster is clean or not
 	_, isClean, err := cephClient.IsClusterClean(r.context.ClusterdContext, request.Name)
 	if err != nil {
-		logger.Errorf("failed to get cephCluster status %+v", err)
 		maxUnavailable := int32(0)
 		mdb.Spec.MaxUnavailable = &maxUnavailable
 		updateErr := r.client.Update(context.TODO(), mdb)
-		if err != nil {
-			logger.Errorf("failed to update mdb %+v", err)
-			return reconcile.Result{}, updateErr
+		if updateErr != nil {
+			return reconcile.Result{}, fmt.Errorf("failed to update mdb %s: %+v", mdb.GetName(), err)
```

Comment:
you are checking `updateErr` and logging/returning `err`.  Please call all errors `err` to avoid confusion.

Reply:
yup done